src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoParticipantDataDao.java

```diff
@@ -0,0 +1,114 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBQueryExpression;
+import com.amazonaws.services.dynamodbv2.datamodeling.PaginatedQueryList;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.dynamodbv2.model.ComparisonOperator;
+import com.amazonaws.services.dynamodbv2.model.Condition;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Iterables;
+import org.sagebionetworks.bridge.BridgeUtils;
+import org.sagebionetworks.bridge.dao.ParticipantDataDao;
+import org.sagebionetworks.bridge.models.ForwardCursorPagedResourceList;
+import org.sagebionetworks.bridge.models.ParticipantData;
+import org.sagebionetworks.bridge.models.subpopulations.StudyConsent;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.Resource;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+@Component
+public class DynamoParticipantDataDao implements ParticipantDataDao {
+    private DynamoDBMapper mapper;
+
+    @Resource(name = "participantDataMapper")
+    final void setParticipantStateMapper(DynamoDBMapper participantStateMapper) {
+        this.mapper = participantStateMapper;
+    }
+
+    public ForwardCursorPagedResourceList<ParticipantData> getParticipantData(String userId, String offsetKey, int pageSize) {
+        checkNotNull(userId);
+        checkNotNull(offsetKey);
```

Comment:
offsetKey is not required. If offsetKey isn't specified, you should skip setting the range key condition.